### PR TITLE
[US440024] Add idol-eductionserver helm chart

### PR DESCRIPTION
**Pull request by joe-wood-opentext**
TODO:

- [x] Update the Eduction config once it's available.
- [x] Update release archives, README and `index.yaml` just before merging.
- [x] ~~Update idol-library to 0.7.0~~

### CODE REVIEW

File: helm/idol-eductionserver/README.md.gotmpl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{{- $appVersion := .AppVersion }}
+{{- $versionList := slice (splitList "." $appVersion) 0 2 }}
+
+{{- $docsLink := printf "https://www.microfocus.com/documentation/idol/IDOL_%s/%s_%s_Documentation/Help/" (join "." $versionList ) $component (join "." $versionList) }}
+
+Provides an IDOL Eduction Server deployment.
+
+Depends on connections to:
+
+- IDOL LicenseServer (`licenseServerHostname`)
```
Comment (christopher-rayson-mf) [inline helm/idol-eductionserver/README.md.gotmpl:16]: Not sure whether this means we should be including the `idol-licenseserver` subchart as a dependency — @daniel-lau-mf, does this add anything, or is defining the licenseserver hostname sufficient?

Comment (daniel-lau-mf) [reply helm/idol-eductionserver/README.md.gotmpl:16]: If the specified `licenseServerHostname` is resolvable by the container then that's sufficient, and doesn't really need a subchart dependency (or it should be optional if it's there).

File: helm/idol-eductionserver/resources/eductionserver.cfg [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+[QueryRole] < "/eductionserver/cfg/original.eductionserver.cfg" [QueryRole]
+Clients=*
+
+[Logging] < "/eductionserver/cfg/original.eductionserver.cfg" [Logging]
+LogDirectory=/idol-logs
+LogArchiveDirectory=/idol-logs/archive
+LogEcho=TRUE
+
+[Eduction]
+ResourceFiles=/eductionserver/grammars/internet.ejr
```
Comment (christopher-rayson-mf) [inline helm/idol-eductionserver/resources/eductionserver.cfg:35]: I can't see this file in the `eductionserver:24.1` image;
```
[idoluser@8a29fe76ce23 eductionserver]$ ls /eductionserver/grammars
ls: cannot access '/eductionserver/grammars': No such file or directory
```
Are we supposed to be getting the grammar into the container, somehow?

Comment (joe-wood-opentext) [reply helm/idol-eductionserver/resources/eductionserver.cfg:35]: Yeah this was just for my local testing, I've got a config map locally to get this in. Will this section once Hasan is done with the pii config and James has updated the image with the grammars/scripts.

**Review by christopher-rayson-mf — APPROVED** [review]

**Pull request merged**